ID/src/main/kotlin/screen/UserScreen.kt

```diff
@@ -35,4 +35,19 @@ class UserScreen {
             }
         }
     }
+
+    fun insertUserOperationNonRecursive(file: File) {
+        print("Put 'q' to quit and put any keys to continue insert user: ")
```

Comment:
In print, message.
Put any keys to continue to insert user, it should change to put any keys to back to main menu.